batnode.js

```diff
@@ -216,10 +216,10 @@ class BatNode {
     let manifest = fileUtils.loadManifest(manifestFilePath);
     const distinctShards = fileUtils.getArrayOfShards(manifestFilePath)
     const fileName = manifest.fileName;
-    this.retrieveSingleCopy(distinctShards, manifest.chunks, fileName, manifestFilePath, distinctIdx, copyIdx)
```

Comment:
I see that `manfestFilePath` was changed to `manifest`, but I liked out the former parameter name was a bit clearer in terms of what value it represents. Otherwise, it's a bit harder to tell if it's manifest contents, meta data, etc. especially without parameter documentation. If you'd like something shorter than `manfestFilePath`, how about `manifestPath`?

Reply:
Since I was trying to read the manifest file in JSON format so would it be better to re-name it as manifestJson as you suggest in below comment?

Reply:
Ah, I see what's going on now. Ignore my first suggestion, and yes, `manifestJson`, `manifestData`, etc. makes more sense like you mentioned.